regression/goto-analyzer/command_line_02/main.c

```diff
@@ -0,0 +1,16 @@
+#include <assert.h>
```

Comment:
Since this test file appears to be identical to the one in command_line_01, why not just have a single file, and use two `.desc` files in the one directory?

Reply:
Fair point, will do in future.  Don't particularly want to change this right now as a hassle getting this back into the right commit.